api_app/analyzers_manager/models.py

```diff
@@ -0,0 +1,128 @@
+from django.db import models
+from django.contrib.postgres import fields as postgres_fields
+from cache_memoize import cache_memoize
+
+from intel_owl.secrets import get_secret
+
+
+class Analyzer(models.Model):
+    TYPE_CHOICES = (
+        ("file_analyzer", "file_analyzer"),
+        ("observable_analyzer", "observable_analyzer"),
+    )
+    QUEUE_CHOICES = (
+        ("default", "default"),
+        ("long", "long"),
+        ("local", "local"),
+    )
+    HASH_CHOICES = (("md5", "md5"), ("sha256", "sha256"))
+
+    name = models.CharField(max_length=128, blank=False, null=False, unique=True)
+    analyzer_type = models.CharField(
+        max_length=50,
+        choices=TYPE_CHOICES,
+    )
+    disabled = models.BooleanField(default=False, blank=False)
+    description = models.TextField()
+    python_module = models.CharField(max_length=128, blank=False, null=False)
+    supported_filetypes = postgres_fields.ArrayField(
+        models.CharField(default=list, max_length=50, blank=True, null=True)
+    )
+    not_supported_filetypes = postgres_fields.ArrayField(
+        models.CharField(default=list, max_length=50, blank=True, null=True)
+    )
+    run_hash = models.BooleanField(default=False, blank=True, null=True)
+    run_hash_type = models.CharField(
+        max_length=50,
+        blank=True,
+        null=True,
+        choices=HASH_CHOICES,
+        default="md5",
+    )
+    observable_supported = postgres_fields.ArrayField(
+        models.CharField(default=list, max_length=50, blank=True, null=True)
+    )
+    leaks_info = models.BooleanField(default=False, blank=True, null=True)
+    external_service = models.BooleanField(default=False, blank=True, null=True)
+    queue = models.CharField(max_length=50, choices=QUEUE_CHOICES, default="default")
+    soft_time_limit = models.IntegerField(default=300)
+    additional_config_params = models.JSONField(default=dict, blank=True, null=True)
```

Comment:
I feel there is no use in keeping `blank` and `null` when you have set a default value.